src/python/grpcio_tests/tests/fork/_fork_interop_test.py

```diff
@@ -131,25 +167,60 @@ def testInProgressBidiNewChannelBlockingCall(self):
 
     def tearDown(self):
         self._server_process.kill()
+        for stream in self._streams:
+            stream.close()
+
+    def _print_backtraces(self, pid):
+        cmd = [
+            "gdb",
+            "-ex",
+            "set confirm off",
+            "-ex",
+            "echo attaching",
+            "-ex",
+            "attach {}".format(pid),
+            "-ex",
+            "echo print_backtrace",
+            "-ex",
+            "thread apply all bt",
+            "-ex",
+            "echo printed_backtrace",
+            "-ex",
+            "quit",
+        ]
+        streams = tuple(tempfile.TemporaryFile() for _ in range(2))
+        sys.stderr.write("Invoking gdb\n")
+        sys.stderr.flush()
+        process = subprocess.Popen(cmd, stdout=streams[0], stderr=streams[1])
+        try:
+            process.wait(timeout=_GDB_TIMEOUT_S)
+        except subprocess.TimeoutExpired:
+            sys.stderr.write("gdb stacktrace generation timed out.\n")
+        finally:
+            for stream_name, stream in zip(("STDOUT", "STDERR"), streams):
```

Comment:
[Minor] Can we modify `_dump_streams` a little bit so that can be used here too?

Reply:
Indeed we can. Nice catch.